.gitignore

```diff
@@ -15,5 +15,6 @@
 /libpeerconnection.log
 npm-debug.log
 testem.log
+.idea
```

Comment:
agreed, any files automatically added by editors are a mistake and won't be merged